Makefile

```diff
@@ -0,0 +1,46 @@
+SHELL:=/bin/sh
```

Comment:
Any chance you can move Makefile to separate folder? If CMake build is done in root dir, it will overwrite this.

Reply:
Yups coming in a few minutes